app/views/pages/api_release_notes.html.haml

```diff
@@ -9,7 +9,71 @@
 
 %section.api-documentation
   %hr
-  %h2 Date TBC
+  %h2 17th September 2020
+  %ul.list-bullet
+    %li
+      New miscellaneous fee types
+      %br/
+      %br/
+      Three new fee types have been added for use on claims 
+      %br
+      %br
+      These will be available for claims with representation orders issued on or after the CLAR implementation date (17th September 2020).
+      %br
+      %br
+      The fee types will be available for AGFS final, supplementary and hardship claims (Paper Heavy and Unused Materials) and LGFS Final and Hardship claims (Unused Materials).
+      %br
+      %br
+      %table
+        %caption
+          New fee types.
+        %thead
+          %tr
+            %th
+              Status
+            %th
+              Unique code
+            %th
+              Description
+            %th
+              Applicability
```

Comment:
probably worth having an extra common on validations - to mention

- unused material upto... quantity of 1 only (may default to it eventually)
- unused material over... quantity of 3+ only
- both only applicable to graduated fee case types (not fixed fee case type such as "appeal against sentence")

- paper heavy case invalid for offence bands 1, 6 and 9 (i.e. could move your *note into thise new column)

Reply:
but other than that i think this looks good